packages/dd-trace/test/platform/node/index.spec.js

```diff
@@ -56,71 +56,32 @@ describe('Platform', () => {
       })
     })
 
-    describe('id', () => {
-      let id
+    describe('crypto', () => {
+      let crypto
       let randomBytes
+      let buffer
 
       beforeEach(() => {
-        const seed = Buffer.alloc(4)
+        buffer = Buffer.alloc(4)
 
-        seed.writeUInt32BE(0xFF000000)
+        buffer.writeUInt32BE(0xabcd1234)
```

Comment:
Can we replace this allocation with a `Buffer.from('abcd1234', 'hex')`? I'm not really leaning one way or the other on this one.